Ports/doom/package.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env -S bash ../.port_include.sh
-port=doom
-workdir=SerenityDOOM-master
-version=git
-files="https://github.com/SerenityPorts/SerenityDOOM/archive/master.tar.gz doom-git.tar.gz"
-makeopts=("-C" "doomgeneric/")
-installopts=("-C" "doomgeneric/")
-launcher_name=Doom
-launcher_category=Games
-launcher_command=doom
+port='doom'
+commit_hash='613f870b6fa83ede448a247de5a2571092fa729d'
+workdir="doomgeneric-${commit_hash}"
+version='git'
+files="https://github.com/ozkl/doomgeneric/archive/${commit_hash}.tar.gz doom-git.tar.gz fabc3e41aca92f58dfdd284754891c17875ec8c995948b49396ead6bc05b8676"
+depends=("SDL2" "SDL2_mixer")
```

Comment:
Please 1) use single quotes for non-interpolated strings and 2) put each array entry on its own line. Same for the other arrays below.

Reply:
Done - I switched to using the long option names for `make` because I think that's more readable than splitting the arguments over 4 lines.